pkgs/applications/audio/grandorgue/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchsvn, cmake, pkg-config, gcc, pkgconfig, fftwFloat, alsaLib
+, zlib, wavpack, wxGTK31, udev, jackaudioSupport ? false, libjack2
+, includeDemo ? true }:
+
+stdenv.mkDerivation rec {
+  pname = "grandorgue";
+  rev = "2333";
+  version = "0.3.1-r${rev}";
+  src = fetchsvn {
+    url = "https://svn.code.sf.net/p/ourorgan/svn/trunk";
+    inherit rev;
+    sha256 = "0xzjdc2g4gja2lpmn21xhdskv43qpbpzkbb05jfqv6ma2zwffzz1";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config gcc ];
```

Comment:
I think `gcc` is already provided by `stdenv`.

Reply:
Thanks for the feedback, I removed `gcc`.

Concerning the warning: I don't see that warning on my NixOS machine, but on ArchLinux (on different hardware), it appears too. I doesn't seem to have any consequences, the program works just fine. I think I'll just use the package, and try to fix problems as they come along :-)